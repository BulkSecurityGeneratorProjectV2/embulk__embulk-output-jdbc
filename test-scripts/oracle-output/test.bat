sqlplus TEST_USER/XXXXXXXX@localhost/TESTDB @test.sql

CALL embulk run test.yml

sqlplus TEST_USER/XXXXXXXX@localhost/TESTDB @select.sql

sed -e "s/ \+/ /g" data/temp.txt > data/temp_trimmed.txt
echo "diff data/test_expected.txt data/temp_trimmed.txt"
diff data/test_expected.txt data/temp_trimmed.txt

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-output-oracle test FAILED!")
