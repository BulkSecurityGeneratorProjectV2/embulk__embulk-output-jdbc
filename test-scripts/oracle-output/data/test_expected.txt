
 ID NUM STR VARSTR DT 
---------- ---------- ---------------- ---------------- -------- 
DTTM0 
--------------------------------------------------------------------------- 
DTTM3 
--------------------------------------------------------------------------- 
 1 123.4 test1 TEST1 15-04-23 
15-04-23 16:02:03.000000 
15-04-23 16:02:03.123000 
 
 2 1234567890 test9999 TEST9999 15-12-30 
15-12-31 14:59:59.000000 
15-12-31 14:59:59.999000 

 ID NUM STR VARSTR DT 
---------- ---------- ---------------- ---------------- -------- 
DTTM0 
--------------------------------------------------------------------------- 
DTTM3 
--------------------------------------------------------------------------- 
 
 3 
 
 
 

